samples/tab-user-profile/ts/Manifest/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.16",
+  "version": "1.2.0",
+  "id": "{{Manifest-Id}}",
+  "packageName": "com.teams.tabuserprofile",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "tab-user-profile",
+    "full": "tab-user-profile"
+  },
+  "description": {
+    "short": "Tab User Profile",
```

Comment:
User profile in Outlook using personal tab app.

Reply:
Fixed